### ISSUE TITLE
[CoE Starter Kit - BUG] your issue

### ISSUE THREAD

**Issue by highsmithjd** [microsoft/coe-starter-kit#1017]
### Describe the issue

I'm trying to walk through the CoE steps and got this error on the first flow. I'm a global admin with a Power Apps Per User license

Your flow could not turn on: CLEANUP - Admin | Sync Template v3 (Check Deleted). Error: The caller with object id '18bff0ec-1f2c-4fbb-aae9-0aeb39ecce0b' does not have permission for connection '/providers/Microsoft.PowerApps/apis/shared_commondataservice/connections/867d43fddc824dfdb89f0f9c7e53efde' under Api 'shared_commondataservice'.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

2.2.20210707.3

### What app or flow are you having the issue with?

CLEANUP - Admin | Sync Template v3 (Check Deleted)

**Comment by manuelap-msft** [comment]
Please try the steps in #947 and see if that resolves your issue.

**Comment by highsmithjd** [comment]
That worked! I had the person who imported the CoE solution, run the Flow. What happens 3 months down the road if he decides to quit and his account is disabled? Do I (or someone) have to reset all the connections?

**Comment by JeneferM-MSFT** [comment]
That is correct. Today connection references are not co-auth friendly and you would need to take ownership of them in the same manner.

**Comment by JeneferM-MSFT** [comment]
thanks for using CoE!
Duplicate of #947

**Comment by highsmithjd** [comment]
Changing their password won't affect the connections will it?

**Comment by manuelap-msft** [comment]
Changing passwords will not impact connections, but if your Azure Active Directory admin enforces new policies (such as multi factor authentication or conditional access policies) that may force them to have to authenticate the connections again.
